src/pages/History.tsx

```diff
@@ -77,6 +79,7 @@ const History = () => {
     return (
         <div id="history">
             <div class="frame">
+                <SettingsCog />
```

Comment:
i wouldnt put it on refund / history as long as there's no settings affecting those pages

Reply:
there may be in the future, so i put it on all pages where the box is.

Reply:
of course, but there arent any yet. so id say only add id there when theres relevant settings